OemPkg/OemPkg.dsc

```diff
@@ -85,6 +85,9 @@
   ConfigVariableListLib|SetupDataPkg/Library/ConfigVariableListLib/ConfigVariableListLib.inf
   ActiveProfileIndexSelectorLib|OemPkg/Library/ActiveProfileIndexSelectorPcdLib/ActiveProfileIndexSelectorPcdLib.inf
 
+  # Add support for GCC stack protector
+  NULL|MdePkg/Library/BaseStackCheckLib/BaseStackCheckLib.inf

```

Comment:
I think we should put this under `[LibraryClasses.AARCH64]`

Reply:
Is it not valid for x86?

Reply:
This stack cookie only has implementation for GCC (I think even that implementation is a pseudo one without legit random number support).

The x64 stack cookie related symbols for VS builds are injected through `MdePkg/Library/BaseBinSecurityLibRng/BaseBinSecurityLibRng.inf`.

Reply:
Right, I'm thinking that x64 builds on GCC may be broken?

Reply:
Ah, yeah, that is set here: https://github.com/microsoft/mu_basecore/blob/347cce4949a5073b284d7d73d307797659033ff0/BaseTools/Conf/tools_def.template#L2075, where x86 builds disabled stack guards with GCC.